onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1290,7 +1290,8 @@ activation and leaky_relu_alpha.)DOC")
                                     "C",
                                     "Input tensor C. "
                                     "The shape of C should be unidirectional broadcastable to (M, N).",
-                                    "T")
+                                    "T",
+                                    OpSchema::Optional)
```

Comment:
Why is this schema change needed? Will this impact forwards or backwards compatibility of models?

Reply:
This is a bug. No, it won't reduce ORT's capability; it only allows more scenarios.